### PR TITLE
[FIX] remove hardcoded table name

### PR DESCRIPTION
**Pull request by nvn-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tivisse** [comment]
Ok it's prettier, but why is it really needed in a stable release ? 

Does it fix something ?

**Comment by nvn-odoo** [comment]
Hi i dont think it solve something. But I guess it is easier when there is a model rename. However. I saw the commit message is not correct. I will fix it tomorrow.I am in holliday today. Thank you for fast review

File: addons/hr_timesheet_attendance/report/hr_timesheet_attendance_report.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class TimesheetAttendance(models.Model):
 
     @api.model_cr
     def init(self):
-        self._cr.execute("DROP VIEW IF EXISTS hr_timesheet_attendance_report")
+        self._cr.execute("DROP VIEW IF EXISTS %s" % self._table)
```
Comment (mreficent) [inline addons/hr_timesheet_attendance/report/hr_timesheet_attendance_report.py]: why not call `drop_view_if_exists(self._table)` from `odoo.tools.sql`?

Comment (nvn-odoo) [reply addons/hr_timesheet_attendance/report/hr_timesheet_attendance_report.py]: Thank you, good idea, now it is done

**Comment by nvn-odoo** [comment]
@tivisse Hi, indeed it doesn't fix stuff but I think it is better to align the way the view is created and dropped.
also fixed the commit message. For me, sounds ok of for merge
Have a great day

**Comment by tivisse** [comment]
You can improve this is master is you wish. We should avoid noise on a stable release.

See: https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean

**Comment by nvn-odoo** [comment]
thank you for the explanation.
Have a great day

**Comment by nvn-odoo** [comment]
@fmdl , as promised, here is the created PR.
regards

**Pull request closed without merging**